src/app/app.component.ts

```diff
@@ -1,21 +1,32 @@
-import { Component, Inject, OnInit, OnDestroy } from '@angular/core';
-import { Router, Route, NavigationEnd } from '@angular/router';
-import { NavigationLinks, AppData } from './app.model';
+import { Component, Inject, OnDestroy, OnInit } from '@angular/core';
+import { NavigationEnd, Route, Router } from '@angular/router';
 import { Subscription } from 'rxjs';
 
+import { AppConfig } from './app-config.interface';
+import { AppData, NavigationLinks } from './app.model';
+
 @Component({
     selector: 'ui-jar-app',
     template: `
         <div class="top-nav-bar">
             <div class="app-title">
-                <a routerLink="/">UI-jar<span>@</span></a>
+                <a routerLink="/" [innerHTML]="config.title || 'UI-jar <span>@</span>'"></a>
             </div>
             <button class="nav-burger-btn" (click)="toggleNavigation()">
                 <span>-</span>
                 <span>-</span>
                 <span>-</span>
             </button>
-            <a href="//github.com/ui-jar/ui-jar" target="_blank" class="project-link">GitHub</a>
+            <a
+                [href]="config.project ? config.project.repository : '//github.com/ui-jar/ui-jar'"
+                target="_blank"
+                class="project-link"
+                [innerHTML]="config.project?.repositoryTitle || 'GitHub'"></a>
+        </div>
+        <div>
```

Comment:
Is this used (div and content)? I couldn't see it when I was trying - maybe I did something wrong?

Reply:
Was my bad, was using it to debbug the config injection and see which one is the best solution hehe